linked_list/linked_list.py

```diff
@@ -23,13 +23,46 @@ def __init__(self):
     self.tail = None
 
   def add_to_tail(self, value):
-    pass
+    new_node = Node(value)
+    if self.head is None:
+      self.head=new_node
+      self.tail=new_node
+    else:
+      self.tail.next_node=Node(value)
```

Comment:
maybe here you set the current tail's next reference to our new node using set_next method, which can take be passed new_node. Then on line 32 set the list's tail reference to new_node